package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@archwayhq/arch3",
-  "version": "0.1.0",
+  "name": "arch3-root",
   "description": "The all-in-one library for your awesome Archway dApp",
   "homepage": "https://docs.archway.io",
   "repository": "github:archway-network/arch3.js",
   "keywords": [
     "archway",
     "cosmos",
+    "cosmjs",
     "cosmosjs",
```

Comment:
interesting that we have `cosmjs` and `cosmosjs`. Creative names.

Do we need both? Not really familiar with them

Reply:
`cosmjs` is the correct library name, but I saw that other projects use `cosmosjs` as a tag, so I kept both only for SEO.